backend/Models/Database/ConservationStatuses.cs

```diff
@@ -0,0 +1,9 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class ConservationStatuses
```

Comment:
As before, name in the singular
```suggestion
    public class ConservationStatus
```